### PR TITLE
Fix HBase filter pushdown

### PR DESCRIPTION
**Pull request by mbautin**

### CODE REVIEW

**Comment by mbautin** [comment]
@rxin good point, submitted a pull request.

**Comment by rxin** [comment]
Thanks - let me know if you have other things like this too :)

File: src/main/scala/shark/execution/TableScanOperator.scala [hunk 1]
```diff
@@ -248,6 +249,11 @@ class TableScanOperator extends TopOperator[HiveTableScanOperator] with HiveTopO
     if (tableDesc != null) {
       Utilities.copyTableJobPropertiesToConf(tableDesc, conf)
     }
+    new HiveInputFormat() {
+      def doPushFilters() {
+        pushFilters(conf, hiveOp)
+      }
+    }.doPushFilters()
```
Comment (rxin) [inline src/main/scala/shark/execution/TableScanOperator.scala:256]: What is the difference between this and directly calling 

pushFilters(conf, hiveOp)

?

**Comment by rxin** [comment]
Ah never mind my question. You need an object to get access to the push filters method.

**Pull request merged**